src/utils/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prefer-rest-params */
```

Comment:
try and figure out why eslint is showing this error. You should not use `arguments`.

Reply:
@bru no need for this in the new PR commit